src/ScenarioUtils/ScenarioUtils.js

```diff
@@ -1,7 +1,16 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
 
-const scenarioExists = (scenarioName, scenarioTree) => {
+const scenarioExistsInList = (scenarioName, scenarioList) => {
+  for (const scenario of scenarioList) {
```

Comment:
Should be better to use Array.find() function instead of a for loop?

Reply:
Done, thanks :+1: